morello/morello.md

```diff
@@ -117,87 +110,76 @@ reserved.
-                                                                                                      C Language Extensions
+[//]: # (morello-table-references)
 
-  [ACLE](https://developer.arm.com/documentation/101028/latest)      Document number: 101028          Arm C Language Extensions
-
-  [CHERI](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-947.pdf)   UCAM-CL-TR-947, SSN 1476-2986    CHERI C/C++ Programming Guide
-  ------------------------------------------------------------------ -------------------------------- -----------------------------
-
-  : :align: left
-:::
+| Ref                                                              |  URL or other reference        |  Title                         |
+| :---                                                             |  :---                          |  :---                          |
+| [ACLE-morello](http://github.com/arm-software/acle/morello)      |  This document                 |  Morello Supplement to the Arm |
+|                                                                  |                                |  C Language Extensions         |
+| [ACLE](https://developer.arm.com/documentation/101028/latest)    |  Document number: 101028       |  Arm C Language Extensions     |
+| [CHERI](https://www.cl.cam.ac.uk/techreports/UCAM-CL-TR-947.pdf) |  UCAM-CL-TR-947, SSN 1476-2986 |  CHERI C/C++ Programming Guide |
 
 ### Terms & Abbreviations
 
-Capability
+_**Capability**_
 
-:   The capability data type is an unforgeable token of authority which
-    provides a foundation for fine grained memory protection and strong
-    compartmentalisation.
+|   The capability data type is an unforgeable token of authority which
+|   provides a foundation for fine grained memory protection and strong
+|   compartmentalisation.
 
-Permissions
+_**Permissions**_
 
-:   The permissions mask controls how the capability can be used - for
-    example, by authorizing the loading and storing of data and/or
-    capabilities.
+|   The permissions mask controls how the capability can be used - for
+|   example, by authorizing the loading and storing of data and/or
+|   capabilities.
 
-Deriving a capability
+_**Deriving a capability**_
 
-:   A capability value CV2 is said to be derived from a capability value
-    CV1 when CV2 is a copy of CV1 with optionally removed permissions
-    and/or optionally narrowed bounds (base increased or limit reduced).
+|   A capability value CV2 is said to be derived from a capability value
+|   CV1 when CV2 is a copy of CV1 with optionally removed permissions
+|   and/or optionally narrowed bounds (base increased or limit reduced).
 
-Sealing a capability
+_**Sealing a capability**_
 
-:   When a capability is sealed it cannot be modified or dereferenced,
-    but it can be used to implement opaque pointer types.
+|   When a capability is sealed it cannot be modified or dereferenced,
+|   but it can be used to implement opaque pointer types.
```

Comment:
Too bad this does not support definition list like pandoc markdown... https://pandoc.org/demo/example19/Extension-definition_005flists.html#Extension-definition_005flists